app/models/like.rb

```diff
@@ -1,5 +1,5 @@
 class Like < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :user, class_name: 'User'
   belongs_to :post
```

Comment:
- [ ] Your code says `Like` belongs to `user` but your migration file for creating `likes` references `author_id`. I would advise you to reference `author` in your migration and then in your model file, `Like` should belong to `author` instead of `user`. I won't be able to like a post if this change is not made and that is the reason why I am making it a required change. 👍 

- [ ] The way you defined the relationship between `Like` and `Post` is alright. But the migration file references `post_id` instead of `post`. All you need to do is `t.references :post, foreign_key: true, null: false` in the column that references post. 👍